azure-pipelines.yml

```diff
@@ -9,32 +9,27 @@ trigger:
     - "*"
 
 jobs:
-- job: build
-  displayName: "Build"
-
+- job: build8
```

Comment:
This file is now very dense - please space it out a little more.

Reply:
Done: added a blank line between jobs, and between steps and configuration for each job.